gustaf/vertices.py

```diff
@@ -10,9 +10,9 @@
 from gustaf import settings
 from gustaf import utils
 from gustaf import show
-from gustaf._base import GustavBase
+from gustaf._base import GustafBase
 
-class Vertices(GustavBase):
+class Vertices(GustafBase):
```

Comment:
`gustav`, you were awesome. You are awesome. You were the best geometry butler I've ever had. Thank you so much for all the amazing work you've done. I am happy you get to retire finally! I will be visiting you. We will be visiting you. Look #8, #15, it is already happening! Call me, I will take you to RAM whenever you want.  Love, J

Reply:
👋